src/main/java/org/orecruncher/dsurround/processing/BiomeSoundHandler.java

```diff
@@ -26,7 +26,7 @@
     public static final int MOOD_SOUND_MAX_RANGE = 16;
 
     // Reusable map for biome acoustic work
-    private static final Object2FloatOpenHashMap<ISoundFactory> WORK_MAP = new Object2FloatOpenHashMap<>(8, 1F);
+    private static final Object2FloatOpenHashMap<ISoundFactory> WORK_MAP = new Object2FloatOpenHashMap<>(8, 0.5F);
```

Comment:
Why was this changed to `0.5F`?

Reply:
Because the constructor of `Object2FloatOpenHashMap` expects that float to be `> 0,0 && < 1.0` and will throw a Exception when `1.0F` is passed. I will, however, probably replace the constant with a static field from fastutil's `Hash` class, which defines `0.75f` as the default load factor and `0.5f` for a small but quick map.

I don't know why this changed while switching to 1.18 or it mightve been broken before too.